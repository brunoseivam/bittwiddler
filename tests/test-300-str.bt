main {
    var hello = "Hello, ";
    var world = "world!";

    var hello_world = hello + world;

    emit("\"{hello}\" + \"{world}\" = \"{hello_world}\"\n");

    var one = "1";
    var two = "2";
    var three = "3";

    var one_two_three = one + ":" + two + ":" + three;

    emit("{one_two_three}\n");

    var a = "127";
    var b = "0";
    var c = "1";
    var home = a + "." + b + "." + b + "." + c;

    emit("home={home}\n");
}
