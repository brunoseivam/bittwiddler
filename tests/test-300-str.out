"Hello, " + "world!" = "Hello, world!"
1:2:3
home=127.0.0.1
